org/party.go

```diff
@@ -122,14 +123,15 @@ type Website struct {
 // The definition found here is based on the details required for spain.
 // If your country requires additional fields, please let us know.
 type Registration struct {
-	UUID    *uuid.UUID `json:"uuid,omitempty" jsonschema:"title=UUID"`
-	Office  string     `json:"office,omitempty" jsonschema:"title=Office"`
-	Book    string     `json:"book,omitempty" jsonschema:"title=Book"`
-	Volume  string     `json:"volume,omitempty" jsonschema:"title=Volume"`
-	Sheet   string     `json:"sheet,omitempty" jsonschema:"title=Sheet"`
-	Section string     `json:"section,omitempty" jsonschema:"title=Section"`
-	Page    string     `json:"page,omitempty" jsonschema:"title=Page"`
-	Entry   string     `json:"entry,omitempty" jsonschema:"title=Entry"`
+	UUID         *uuid.UUID `json:"uuid,omitempty" jsonschema:"title=UUID"`
+	ShareCapital num.Amount `json:"social_capital,omitempty" jsonschema:"title=Social Capital"`
```

Comment:
Go property name should match json tag. Perhaps just `capital` is enough given the context, certainly shorter which is nice. We shouldn't use a `num.Amount` for this value. There are two issues here: the currency of the document might not match, and a pointer should be used to avoid all invoices being generated with a zero value.